src/app/models/Statistics.ts

```diff
@@ -0,0 +1,105 @@
+import { AUDIO_RECORDING, SHALLOW_AUDIO_EVENT } from "@baw-api/ServiceTokens";
+import { statisticsMenuItem } from "@components/statistics/statistics.menus";
+import { DateTimeTimezone, Id, Ids } from "@interfaces/apiInterfaces";
+import { AbstractModel } from "@models/AbstractModel";
+import { hasMany } from "@models/AssociationDecorators";
+import {
+  bawCollection,
+  bawDateTime,
+  bawDuration,
+} from "@models/AttributeDecorators";
+import { AudioEvent } from "@models/AudioEvent";
+import { AudioRecording } from "@models/AudioRecording";
+import { Duration } from "luxon";
+
+export interface IStatisticsSummary {
+  usersOnline: number;
+  usersTotal: number;
+  onlineWindowStart: DateTimeTimezone | string;
+  projectsTotal: number;
+  regionsTotal: number;
+  sitesTotal: number;
+  annotationsTotal: number;
+  annotationsTotalDuration: Duration | number;
+  annotationsRecent: number;
+  audioRecordingsTotal: number;
+  audioRecordingsRecent: number;
+  audioRecordingsTotalDuration: Duration | number;
+  audioRecordingsTotalSize: number;
+  tagsTotal: number;
+  tagsAppliedTotal: number;
+  tagsAppliedUniqueTotal: number;
+}
+
+export class StatisticsSummary extends AbstractModel {
+  public readonly kind = "StatisticsSummary";
+  public readonly usersOnline: number;
+  public readonly usersTotal: number;
+  @bawDateTime()
+  public readonly onlineWindowStart: DateTimeTimezone;
+  public readonly projectsTotal: number;
+  public readonly regionsTotal: number;
+  public readonly sitesTotal: number;
+  public readonly annotationsTotal: number;
+  @bawDuration()
+  public readonly annotationsTotalDuration: Duration;
+  public readonly annotationsRecent: number;
+  public readonly audioRecordingsTotal: number;
+  public readonly audioRecordingsRecent: number;
+  @bawDuration()
+  public readonly audioRecordingsTotalDuration: Duration;
+  public readonly audioRecordingsTotalSize: number;
+  public readonly tagsTotal: number;
+  public readonly tagsAppliedTotal: number;
+  public readonly tagsAppliedUniqueTotal: number;
+
+  public get viewUrl(): string {
+    throw Error();
+  }
+}
+
+export interface IStatisticsRecent {
+  audioRecordingIds: Ids | Id[];
+  audioEventIds: Ids | Id[];
+}
+
+export class StatisticsRecent extends AbstractModel {
+  public readonly kind = "StatisticsRecent";
+  @bawCollection()
+  public readonly audioRecordingIds: Ids;
+  @bawCollection()
+  public readonly audioEventIds: Ids;
+  @hasMany<StatisticsRecent, AudioRecording>(
+    AUDIO_RECORDING,
+    "audioRecordingIds"
+  )
+  public audioRecordings: AudioRecording[];
+  @hasMany<StatisticsRecent, AudioEvent>(SHALLOW_AUDIO_EVENT, "audioEventIds")
+  public audioEvents: AudioEvent[];
+
+  public get viewUrl(): string {
+    throw Error();
+  }
+}
+
+export interface IStatistics {
+  summary: StatisticsSummary | IStatisticsSummary;
+  recent: StatisticsRecent | IStatisticsRecent;
+}
+
+export class Statistics extends AbstractModel {
```

Comment:
Hmm, these models are not really descendants of our normal API structure.  It's worth thinking about if AbstractModel is the right base class.

For example, they do not have an `id` field, unlike all other models